plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -105,15 +125,28 @@ public function render_column( $column_id, $order ) {
 	 * @param string    $column_name Identifier for the custom column.
 	 */
 	public function column_default( $order, $column_name ) {
-		/**
-		 * Fires for each custom column in the Custom Order Table in the administrative screen.
-		 *
-		 * @param string    $column_name Identifier for the custom column.
-		 * @param \WC_Order $order       Current WooCommerce order object.
-		 *
-		 * @since 7.0.0
-		 */
-		do_action( "manage_{$this->screen->id}_custom_column", $column_name, $order );
+		if ( has_action( 'woocommerce_' . $this->order_type . '_list_table_custom_column' ) ) {
```

Comment:
won't they both need to be fired? for example, since it only checked for the filter name, an action for one column (say column1) can disable action for another column (say column2), if column1 is registering on `woocommerce_{$this->order_type}_list_table_column` and column2 is registering on `manage_{$this->screen_id}_custom_column`.

Reply:
Yes. I would've preferred we don't have this "generic" version of the hook (instead of the order-type specific one), but given it exists, I agree that it'd be best to fire both just in case. I've done that in cb6a787c59c982c4b95624d314943f703730b72d.